src/routes/upload/_MissionSection.svelte

```diff
@@ -126,7 +117,7 @@
 		})
 			.then(response => {
 				const plural = Object.values(selectedMissions).filter(value => value).length > 1;
-				const word = `Misison${plural ? 's' : ''}`;
+				const word = `Mission${plural ? 's' : ''}`;
```

Comment:
Use util.ts pluralize

Reply:
The pluralize function also states the #. That is not needed. It would actually lead to confusion in this case as a user seeing "3 missions failed to upload" would think that all 3 missions failed instead of a single one. Better error handling from the server to the client should be done.